devices/leedarson.js

```diff
@@ -85,7 +85,7 @@ module.exports = [
         extend: extend.light_onoff_brightness_colortemp(),
     },
     {
-        zigbeeModel: ['ZHA-PIRSensor'],
+        fingerprint: [{modelID: 'ZHA-PIRSensor', manufacturerName: 'Leedarson'}],
```

Comment:
Have you checked with an actual device that it uses `manufacturerName` `Leedarson`?

Reply:
@Koenkk here https://github.com/Koenkk/zigbee2mqtt/issues/3339

Reply:
thanks!